examples/mongoose.test.js

```diff
@@ -2045,4 +2045,26 @@ describe('Mongoose', function() {
     assert.equal(profilingLevel, 'off');
     // acquit:ignore:end
   });
+
+  it('validates-email', async function() {
+    const User = mongoose.model('User', mongoose.Schema({
+      email:{ 
+      type: String,
+      required: true,
+      match: /.+\@.+\..+/,
+      unique: true
+      }
+    }));
+    await User.create([
+        { email: 'gmail@google.com' },
+        { email: 'bill@microsoft.com' },
+        { email: 'test@gmail.com' }
+    ]);
+    console.log(await User.find()); // Prints the information associated with the three emails above.
+    await User.init().then(function() {
```

Comment:
No need for `then()` here. Just `await User.init();` followed by `await User.create(...)`